lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1748,11 +1748,17 @@ void SwiftASTContext::FilterClangImporterOptions(
   // Copy back a filtered version of ExtraArgs.
   std::vector<std::string> orig_args(std::move(extra_args));
   for (auto &arg : orig_args) {
+    StringRef arg_sr(arg);
+    // LLDB wants to control implicit/explicit modules build flags.
+    if (arg_sr == "-fno-implicit-modules" ||
+        arg_sr == "-fno-implicit-module-maps")
+      continue;
```

Comment:
should we put this inside a setting? could it cause issues?

Reply:
If we really need to we can replace with the extra clang flags setting already.

Reply:
So there should be no need to add another setting to control this.

Reply:
I'm worried that by doing this to fix the one bug report, we may create new unforeseen bugs. However, that's entirely speculative on my part. I guess we'll find out whether allowing implicit modules in the presence of explicit modules has any gotchas.

Reply:
I don't think this will do much to fix that specific bug, but I do think that the target's build settings should not control how LLDB imports modules.